great_expectations/util.py

```diff
@@ -61,6 +74,56 @@ def get_project_distribution() -> Union[Distribution, None]:
     return None
 
 
+# Returns the object reference to the currently running function (i.e., the immediate function under execution).
```

Comment:
Very cool!

Reply:
I hope that others will find use for this and the function that gathers the arguments of the presently running method.